package.json

```diff
@@ -9,24 +9,27 @@
     "test:end2end": "mocha test/e2e --timeout 15000",
```

Comment:
test command doesn't generate coverage report, so (I guess) running sonar scan does not generate a report with coverage

Reply:
I would ignore this for now